### ISSUE TITLE
Reported sites unblocked / on hold

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#190]
Not sure why some reported sites are showing as  both unblocked *and* on hold.

If this is right, then the scheduled test presumably shouldn't send a report, and the report (attempt) should be hidden / discarded / removed.

**Comment by dantheta** [comment]
Those are ones that were queued for retesting, and the retest came back as "not blocked".  Behind the scenes, the status of those reports is set to "cancelled".  I've hidden those (and ones which are flagged as system abuse) from the legal blocks page.
